pkg/ccl/testccl/workload/schemachange/schemachange_test: TestWorkload failed

pkg/ccl/testccl/workload/schemachange/schemachange_test.TestWorkload [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/13902447?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/13902447?buildTab=artifacts#/) on master @ [049d54d18aead4c10308ebb9e996451dffe2c9c4](https://github.com/cockroachdb/cockroach/commits/049d54d18aead4c10308ebb9e996451dffe2c9c4):


```
     "INDEX schema_w0_17.table_w0_32@\"table_w0_32_col\n32__w0_35_key\"",
     "INDEX schema_w0_17.table_w0_32@\"table_w0_32_col3\\\\x072__w0_33_key\"",
     "INDEX schema_w0_45.table_w0_49@table_w0_49_col49__w0_52_idx",
     "INDEX schema_w0_45.table_w0_49@\"table_w0_49_c\rol49__w0_62_col49__w0_63_key\"",
     "INDEX schema_w0_45.table_w0_49@table_w0_49_col49__w0_52_key",
     "INDEX schema_w0_45.table_w0_49@\"table_w0_49_col49'_͔_w0_58_col49__w0_60_idx\"",
     "INDEX schema_w0_45.table_w0_49@table_w0_49_col49__w0_57_key",
     "INDEX schema_w0_45.table_w0_49@\"table_w0_49_expr_\\rcol49__w0_51_idx\"",
     "INDEX schema_w0_45.table_w0_49@table_w0_49_col49__w0_56_key"
    ]
   }
  ],
  "previousStatements": [
   "COMMENT ON TABLE schema_w0_17.table_w0_32 IS 'comment from the RSW'"
  ]
 }
}
    schema_change_external_test.go:132: 
        	Error Trace:	pkg/ccl/testccl/workload/schemachange/schemachange_test_test/pkg/ccl/testccl/workload/schemachange/schema_change_external_test.go:132
        	Error:      	Received unexpected error:
        	            	***UNEXPECTED COMMIT ERROR; Received an unexpected commit error: ERROR: error executing PreCommitPhase stage 2 of 2 with 1 MutationType op: persisting catalog mutations: unexpected value: raw_bytes:"\361j\331h\n" timestamp:<wall_time:1707383165331755471 logical:1 >  (SQLSTATE XXUUU)
        	            	(1) forced error mark
        	            	  | "fatal error when running txn"
        	            	  | github.com/cockroachdb/errors/withstack/*withstack.withStack::
        	            	Wraps: (2)
        	            	Wraps: (3) attached stack trace
        	            	  -- stack trace:
        	            	  | github.com/cockroachdb/cockroach/pkg/workload/schemachange.(*schemaChangeWorker).run
        	            	  | 	github.com/cockroachdb/cockroach/pkg/workload/schemachange/schemachange.go:612
        	            	  | pkg/ccl/testccl/workload/schemachange/schemachange_test_test.TestWorkload.TestWorkload.func4.func6
        	            	  | 	pkg/ccl/testccl/workload/schemachange/schemachange_test_test/pkg/ccl/testccl/workload/schemachange/schema_change_external_test.go:121
        	            	  | golang.org/x/sync/errgroup.(*Group).Go.func1
        	            	  | 	golang.org/x/sync/errgroup/external/org_golang_x_sync/errgroup/errgroup.go:75
        	            	  | runtime.goexit
        	            	  | 	src/runtime/asm_amd64.s:1650
        	            	Wraps: (4) ***UNEXPECTED COMMIT ERROR; Received an unexpected commit error
        	            	Wraps: (5) ERROR: error executing PreCommitPhase stage 2 of 2 with 1 MutationType op: persisting catalog mutations: unexpected value: raw_bytes:"\361j\331h\n" timestamp:<wall_time:1707383165331755471 logical:1 >  (SQLSTATE XXUUU)
        	            	Error types: (1) *markers.withMark (2) *schemachange.ErrorState (3) *withstack.withStack (4) *errutil.withPrefix (5) *pgconn.PgError
        	Test:       	TestWorkload
Schema Workload Stats
Total Schema Statements Executed = 61
Total Schema Statements Succeeded = 61
Total Schema Statement Expected Failures = 0
Total Transactions Committed = 28
Total Transactions Rolled Back = 30
Total Transactions Executed = 58
    schema_change_external_test.go:98: backup, tracing data, and system table dumps in /artifacts/tmp/_tmp/fcba020ca7cb44bf8385bd0e13bcbe2d/logTestWorkload3520683626
    panic.go:523: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/fcba020ca7cb44bf8385bd0e13bcbe2d/logTestWorkload3520683626
--- FAIL: TestWorkload (44.70s)
```

Parameters:
 - <code>TAGS=bazel,gss,deadlock</code>
 - <code>stress=true</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #118879 pkg/ccl/testccl/workload/schemachange/schemachange_test: TestWorkload failed [C-test-failure O-robot P-2 T-sql-foundations branch-release-23.1.15-rc]
- #117782 pkg/ccl/testccl/workload/schemachange/schemachange_test: TestWorkload failed [C-test-failure O-robot P-3 T-sql-foundations branch-release-23.2.0-rc]
</p>
</details>
/cc @cockroachdb/sql-foundations
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestWorkload.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>


Jira issue: CRDB-35966

--- a/pkg/workload/schemachange/optype.go
+++ b/pkg/workload/schemachange/optype.go
@@ -279,7 +279,7 @@ var opWeights = []int{
 	alterTableSetColumnDefault:        1,
 	alterTableSetColumnNotNull:        1,
 	alterTypeDropValue:                0, // Disabled and tracked with #114844, #113859, and #115612.
-	commentOn:                         1,
+	commentOn:                         0, // Disabled and tracked with #116795.
 	createFunction:                    1,
 	createIndex:                       1,
 	createSchema:                      1,

